docs/cicd_tools/log.md

```diff
@@ -0,0 +1,61 @@
+# Log module
+
+This module provides functions to help write log messages.
+
+## Definition
+
+The module ID is `log`
+
+All functions exposed by this module will use the namespaced prefix:
+
+```
+cicd::log::
+```
+
+## Usage
+
+Use the `log` id to load the module via the `loader` module, using either the `load_module.sh`
+or the `bootstrap.sh` script
+
+### Enabling debug output
+
+enabling the debug output requires to export the `CICD_LOG_DEBUG` variable with a non-empty value.
+
+```
+export CICD_LOG_DEBUG=1
+```
+
+## Configuration and public functions
+
+### Dependencies
+
+None
+
+### Configuration Variables
+
+- `CICD_LOG_DEBUG` if defined to a non-empty value enables the messages printed through
```

Comment:
Might be more clear as:

Set the `CICD_LOG_DEBUG` variable to any non-empty value to enable debug logging. Debug logging is provided by the `cicd::log::debug` function.

Reply:
Thanks, updated!